manifests/bundle.pp

```diff
@@ -29,8 +30,9 @@
     cwd         => $cwd,
     user        => $user,
     logoutput   => 'on_failure',
-    timeout     => '600',
+    timeout     => $timeout,
     path        => $path,
     unless      => $unless,
+    npm_timeout => $npm_timeout
```

Comment:
This does not exist: https://puppet.com/docs/puppet/latest/type.html#exec-attributes
```suggestion
```